boards/xtensa/intel_adsp_baytrail/Kconfig.defconfig

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_INTEL_ADSP_BAYTRAIL
+
+config BOARD
+	default "intel_adsp_baytrail"
+
+config CAVS_ICTL_0_OFFSET
+	default 6
+config CAVS_ICTL_1_OFFSET
+	default 10
+config CAVS_ICTL_2_OFFSET
+	default 13
+config CAVS_ICTL_3_OFFSET
+	default 16
+
+config MAX_IRQ_PER_AGGREGATOR
+	default 32
+
+config HEAP_MEM_POOL_SIZE
+	default 1024
+	depends on DMA_DW
+
+config DMA_DW
+	default y
+	depends on DMA
+
+config I2S_CAVS
+	default y
+	depends on I2S
+
+config HEAP_MEM_POOL_SIZE
```

Comment:
think this should be combined with the prior definition a few lines above (if we keep this at all)